src/System.Private.CoreLib/src/System/Reflection/Emit/TypeBuilderInstantiation.cs

```diff
@@ -90,11 +90,10 @@ public override Type MakeArrayType(int rank)
             if (rank <= 0)
                 throw new IndexOutOfRangeException();
 
-            string comma = "";
-            for (int i = 1; i < rank; i++)
-                comma += ",";
+            string s = rank == 1 ?
+                "[]" :
```

Comment:
Just noticed I should probably add a check for rank == 2 here; otherwise for rank == 2 I'm actually adding a string allocation that wasn't there before.

Reply:
Actually, scratch that, was misreading it on second read.